ui/app/styles/core/field.scss

```diff
@@ -19,7 +15,12 @@
     background-color: $ui-gray-100;
     cursor: not-allowed;
   }
+
+  &:not(:last-child) {
+    margin-bottom: $size-4;
+  }
```

Comment:
Previously this was applied only to the `field` class. Will it be an issue adding it to `field-body` as well?

Reply:
It's hard to see because of the diff, but it's still only applied to field. 
![image](https://user-images.githubusercontent.com/6618863/232633933-c93e5cd1-ac29-4f12-802a-abea59002e67.png)